sass/components/_timepicker.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .text-primary {
-	color: rgba(255, 255, 255, 1);
+	color: rgba(255, 255, 255, 1) !important;
```

Comment:
Do you really need the `!important` specifier ?

As it should be mostly used as a tool of last resort (since it breaks the principle of *cascading* stylesheet), try to use a more specific selector.

Reply:
`!important` is not necessary here. I will remove it.